chunks/tool-yugabytedb/chunk.yaml

```diff
@@ -11,6 +11,6 @@ variants:
       PYTHON_VERSION: 3.9
   - name: "2.20"
     args:
-      YB_VERSION: 2.20.0.0
-      YB_BUILD: 76
+      YB_VERSION: 2.20.1.3
+      YB_BUILD: 3
```

Comment:
Non-blocking:
How do you determine the value for YB_BUILD? Just curious.

Reply:
It's actually my first time doing this, so mayhaps I could be doing it better, but I just took what was written on their [downloads page](https://download.yugabyte.com/#linux).

Reply:
Ah, nice, I [see now](https://github.com/gitpod-io/workspace-images/blob/ef1c6b1162bb0ac8502728ea50844ecd31b83318/chunks/tool-yugabytedb/Dockerfile#L21). I missed the `b` earlier. :)